### ISSUE TITLE
Packages matrix

### ISSUE THREAD

**Issue by moltar** [moltar/typescript-runtime-type-benchmarks#824]
Create a table / matrix for each package and features it supports.

**Comment by hoeck** [comment]
Could generate that table as part of the benchmark runs and also add it it as filter checkboxes to the interactive graph.

Any ideas on which features are relevant? I'd like to see "uses eval / new Function", "strict validation mode", "safe validation mode", "parsing / transforming".

**Comment by moltar** [comment]
Yeah, that list is pretty much what I was thinking. Maps well to your solution with splitting the suites.

I am also interested in "JSON Schema support", but this is not really a big deal, if it doesn't fit in the model described above.

**Comment by hoeck** [comment]
> Yeah, that list is pretty much what I was thinking. Maps well to your solution with splitting the suites.
> 
> I am also interested in "JSON Schema support", but this is not really a big deal, if it doesn't fit in the model described above.

Yea that definitely fits in, should that be two checkboxes, "Creates JSON Schema from types" and "Creates types from JSON Schema"?

In addition I'd like to add the number of dependencies (:thinking: that could be even scripted using the npm api via libnpm), maybe the time of its last release and typescript version too.
